### ISSUE TITLE
Any plan to support multiple languages searching?

### ISSUE THREAD

**Issue by quickerlab** [withspectrum/spectrum#3197]
Post a non-latin thread, the search engine can't find the thread.
Any plan to support multiple languages searching?
thanks.

**Comment by quickerlab** [comment]
Correct the question.
From "Home" -> "everything" -> "Search for conversations", can get the non-latin result.
From "Home" -> "Community" -> "Search this community",  also can get non-latin result.
From "Community" -> ”Search“ -> "Search all threads in community", no result when searching  non-latin strings.

Does this a bug?

**Comment by brianlovin** [comment]
@quickerlab could you share the search query and the community that can help us track this behavior down?

**Comment by quickerlab** [comment]
@brianlovin , thank you.
Community: https://spectrum.chat/cs304-sustech
Query strings: "规则"
I can find this thread
https://spectrum.chat/?t=65e69f50-0252-44c0-8453-89359511dce8
at "Home" -> "Search everything" and "Home" -> "Search community"
But I can't find the thread at community page -> "Search" -> "Search all threads"

**Comment by brianlovin** [comment]
Yeah, can repro - that's super weird though, thanks for finding this!
